prime-router/src/main/kotlin/azure/observability/event/PerformerSummary.kt

```diff
@@ -0,0 +1,54 @@
+package gov.cdc.prime.router.azure.observability.event
+
+import org.hl7.fhir.r4.model.Base
+import org.hl7.fhir.r4.model.DomainResource
+
+data class PerformerSummary(
+    val performerName: String = UNKNOWN,
+    val performerState: String = UNKNOWN,
+    val performerCLIA: String = UNKNOWN,
+) {
+    companion object {
+        const val UNKNOWN = "Unknown"
+
+        /**
+         * Create an instance of [PerformerSummary] from a [DomainResource]
+         */
+        fun fromPerformer(performer: Base): PerformerSummary {
+            when (performer) {
+                is org.hl7.fhir.r4.model.Practitioner -> {
+                    // Practitioner has a name list and an address list
+                    val performerName = performer.name
+                        ?.firstOrNull()
+                        ?.nameAsSingleString ?: UNKNOWN
+
+                    val performerState = performer.address
+                        ?.firstOrNull()
+                        ?.state ?: UNKNOWN
+
+                    // For Practitioner, identifier list may contain CLIA. You need to find it explicitly.
+                    val performerCLIA = performer.identifier
+                        ?.firstOrNull { it.system?.contains("CLIA", ignoreCase = true) == true }
```

Comment:
Another issue that may also need to wait until next week when Victor is back in. All the Simple Report fhir examples in our repo do not have CLIA under identifier.system but instead under identifier.type.code, example:
```json
  "identifier": [
    {
      "use": "official",
      "type": {
        "coding": [
          {
            "system": "http://terminology.hl7.org/CodeSystem/v2-0301",
            "code": "CLIA"
          }
        ]
      },
      "value": "11D1111111"
    }
  ],
```

Reply:
Today we looked through an example of this in prod and it confirmed that "CLIA" is going to be communicated through `identifier.type.coding.code` and not `identifier.system`.